src/main/java/com/qaprosoft/carina/demo/mobilepractice/gui/pages/ios/LabelMenuPage.java

```diff
@@ -0,0 +1,26 @@
+package com.qaprosoft.carina.demo.mobilepractice.gui.pages.ios;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.foundation.webdriver.locator.ExtendedFindBy;
+import com.qaprosoft.carina.demo.mobilepractice.gui.pages.common.LabelMenuPageBase;
+import org.openqa.selenium.WebDriver;
+
+import java.util.List;
+
+@DeviceType(pageType = DeviceType.Type.IOS_PHONE, parentClass = LabelMenuPageBase.class)
+public class LabelMenuPage extends LabelMenuPageBase {
+
+    @ExtendedFindBy(accessibilityId = "Done")
+    private ExtendedWebElement doneButton;
+
+    @ExtendedFindBy(iosClassChain = "**/XCUIElementTypeCollectionView")
+    private List<ExtendedWebElement> groupList;
```

Comment:
what is this element for? why it's a List?

Reply:
It's a list because you can add more gropus on account configuration from iOS (if you have other emails linked)